### PR TITLE
vm_test: use a simpler smaller VMI

### PR DESCRIPTION
**Pull request by dankenigsberg**
The tests that are affected by the modified `BeforeEach` do not make use of the guest. They can run just as well with much less memory.

/sig code-quality

```release-note
NONE
```

### CODE REVIEW

**Review by fossedihelm — APPROVED** [review]

**Review by brianmcarey — COMMENTED** [review]
Just a heads up - sometimes there are different memory requirements for VMIs in the arm64 e2e tests. 

See for example - https://github.com/kubevirt/kubevirt/pull/11366

The lane is not currently being reported as it was failing and we have been working on getting it passing again so we can make it required. 
In this case - the lane looks ok so this change doesn't seem to negatively affect it- https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/11374/pull-kubevirt-e2e-arm64/1762570301800976384

This extra memory requirement was kind of hidden under NewRandomVMI- https://github.com/kubevirt/kubevirt/blob/9d4a040652074af39d58eeea6a9cf3042b64d285/tests/utils.go#L394C1-L400C3

**Review by enp0s3 — COMMENTED** [review]
/approve

**Comment by enp0s3** [comment]
@dankenigsberg Can you please rebase on top of #11401?

**Comment by dankenigsberg** [comment]
> @dankenigsberg Can you please rebase on top of #11401?

I believe Tide would be doing this automatically.

File: tests/libvmi/factory.go [hunk 1]
```diff
@@ -85,6 +85,23 @@ func NewAlpineWithTestTooling(opts ...Option) *kvirtv1.VirtualMachineInstance {
...
+		opts...)
+	return New(opts...)
+}
+
+func qemuMinimumMemory() string {
+	if isARM64() {
+		// required to start qemu on ARM with UEFI firmware
+		// https://github.com/kubevirt/kubevirt/pull/11366#issuecomment-1970247448
+		const armMinimalBootableMemory = "128Mi"
+		return armMinimalBootableMemory
```
Comment (orenc1) [inline tests/libvmi/factory.go:100]: What is the benefit of defining that constant, whose scope is inside the function, instead of just `return "128Mi"` ?

Comment (dankenigsberg) [reply tests/libvmi/factory.go:100]: I hate it too. It was a means to silence a linter.

**Pull request merged**